app/budget/directives/budgetCosts/budgetCostsPrint/budgetCostsPrint.js

```diff
@@ -0,0 +1,57 @@
+import { toCurrency } from 'shared/helpers/string';
+
+import './budgetCostsPrint.css';
+
+let budgetCostsPrint = function ($rootScope, BudgetActions, TermService) {
+	return {
+		restrict: 'E',
+		template: require('./budgetCostsPrint.html'),
+		replace: true,
+		scope: {
+			state: '<',
+			summary: '<',
+			instructorTypes: '<',
+			selectedBudgetScenario: '<',
+			termNav: '<',
+			scheduleCosts: '<',
+			instructorAssignmentOptions: '<',
+			regularInstructorAssignmentOptions: '<',
+			isLiveDataScenario: '<',
+			lineItems: '<'
+		},
+		link: function (scope){
+			var json = JSON.parse(localStorage.workgroup);
+			scope.workgroupName = json.name;
+
+			scope.allTerms = {
```

Comment:
This should not be hardcoded in the directive, use a helper service to get these values (I think term service may already have this available? unsure).

Reply:
I was already using TermService. Just forgot to delete that variable :)